addons/gdUnit4/src/monitor/GodotGdErrorMonitor.gd

```diff
@@ -14,7 +14,7 @@ var _report_force : bool
 func _init(force := false):
 	super("GodotGdErrorMonitor")
 	_report_force = force
-	_godot_log_file = GdUnitSettings.get_log_path().get_base_dir() + "/godot.log"
+	_godot_log_file = GdUnitSettings.get_log_path()
```

Comment:
please add this test to `res://addons/gdUnit4/test/monitor/GodotGdErrorMonitorTest.gd`

```
func test_integration_test() -> void:
	var monitor := GodotGdErrorMonitor.new(true)
	# no errors reported
	monitor.start()
	monitor.stop()
	assert_array(monitor.reports()).is_empty()
	
	# push error
	monitor.start()
	push_error("Test GodotGdErrorMonitor 'push_error' reporting")
	monitor.stop()
	assert_array(monitor.reports()).is_not_empty()
	if not monitor.reports().is_empty():
		assert_str(monitor.reports()[0].message()).contains("Test GodotGdErrorMonitor 'push_error' reporting")
	else:
		fail("Expect reporting push_error")
```